.gitignore

```diff
@@ -1,4 +1,5 @@
 pear-algorithm
```

Comment:
Delete `pear-algorithm` and ensure that `src/pear_algorithm` hides the contents of the folder and not the folder itself, otherwise we will have submodule issues (if you try to access the folder it will take you to the private Github page hosting the algorithm so we need the folder).  Also, rename the folder to `pear-algorithm` so that its name is consistent with the repos name.